extensions/3DTILES_implicit_tiling/README.md

```diff
@@ -0,0 +1,631 @@
+
+#### Level Offset Fill
+
+The level offset fill describes the state of all tiles in levels before the level offset.
+
+### Bitstream Layout
+
+Each bitstream encodes tile state information using an adaptive linear quadtree or octree (depending on the `tilingScheme`) representation of the hierarchy. Tile states are packed in increasing order of level and Morton index of tiles within that level. Only tiles that subdivide internally will have information about their children present at the next level. This allows for a space efficient representation of sparse hierarchies.
+
+*The following section is non-normative.*
+
+The following example illustrates the usage of these buffers in a sparse quadtree that extends to 2 levels. 
+
+```json
+{
+    "tilingScheme": "quadtree",
+    "subdivision": {
+        "bufferView": 0
+    },
+    "content": {
+        "levelOffset": 2,
+        "levelOffsetFill": 1,
+        "bufferView": 1
+    },
+    "metadata": {
+        "levelOffset": 2,
+        "levelOffsetFill": 0,
+        "bufferView": 0
+    }
+}
+```
+
+Only the information with grey background is present in the bitstream.
+
+![States](figures/states.png)
+
+### Root Tiles
+
+Multiple tilesets using this extension can be combined by using parent tileset referring to one or more external tilesets using `3DTILES_implicit_tiling` as the children of the root tile.
+
+### Jump Buffer
+
+The jump buffer is a buffer created at runtime to allow efficient traversal and tile lookup functions. The jump buffer is a bijection between a tile's Morton index and its index in the level.
+
+## Properties Reference
+
+---------------------------------------
+### 3DTILES_implicit_tiling Tileset JSON extension
+
+Specifies the Tileset JSON properties for the 3DTILES_implicit_tiling.
+
+**Properties**
+
+|   |Type|Description|Required|
+|---|----|-----------|--------|
+|**boundingVolume**|`object`|A bounding volume that encloses the tileset.|☑️ Yes|
+|**tilingScheme**|`string`|A string describing the tiling scheme used within the tileset|☑️ Yes|
+|**subdivision**|`object`|An object containing high level information about the subdivision buffer|☑️ Yes|
+|**content**|`object`|An object containing high level information about the content buffer. This may be omitted if no tiles in the tileset contain content.|No|
+|**metadata**|`object`|An object containing high level information about the metadata buffer. This may be omitted if no tiles in the tileset contain metadata.|No|
+|**bufferViews**|`array`|An array containing typed views into buffers|No|
+|**buffers**|`array`|An array of buffers.|No|
+|**tileExtension**|`string`|The extension applied to each tile in the tileset.|No|
+|**tilesetExtension**|`string`|The extension applied to each implicit external tileset in the tileset.|No|
+
+Additional properties are not allowed.
+
+---
+
+
+### boundingVolume 
+
+A bounding volume that encloses the tileset.  Exactly one `box`, `region` or `cell` property is required.
```

Comment:
These sections should describe *how* the bounding volume gets subdivided for octree and quadtree alongside the diagrams.

Reply:
Added description of how split axes get defined.